lib/events/filelog.go

```diff
@@ -374,7 +374,7 @@ func (l *FileLog) SearchSessionEvents(fromUTC, toUTC time.Time, limit int, order
 
 type searchEventsFilter struct {
 	eventTypes []string
-	condition  EventFieldsCondition
+	condition  utils.FieldsCondition
 }
 
 // Close closes the audit log, which inluces closing all file handles and releasing
```

Comment:
I know you didn't make this typo

```suggestion
// Close closes the audit log, which includes closing all file handles and releasing
```

Reply:
Done, not to worry.

So many typos on the codebase, even with misspell. I wonder if there's a better automated checker somewhere else.